summingbird-storm/src/main/scala/com/twitter/summingbird/storm/spout/RichStormSpout.scala

```diff
@@ -0,0 +1,38 @@
+package com.twitter.summingbird.storm.spout
+
+import backtype.storm.spout.SpoutOutputCollector
+import backtype.storm.task.TopologyContext
+import backtype.storm.topology.{ IRichSpout, OutputFieldsDeclarer }
+import backtype.storm.tuple.Fields
+import com.twitter.summingbird.storm.Constants._
+import com.twitter.tormenta.spout.SpoutProxy
+import java.util
+import java.util.{ List => JList }
+import com.twitter.summingbird.storm.collector.KeyValueOutputCollector
+
+/**
+ * Created by pnaramsetti on 7/19/16.
+ *
+ * This is a spout used when the spout is being followed by summer.
+ * It uses a KeyValueOutputCollector on open.
+ */
+class RichStormSpout(val self: IRichSpout) extends SpoutProxy {
```

Comment:
this is not such a descriptive name. Can you call it `MapperSpout` or something? Maybe `DirectToSummerSpout`?